nodejs/tsconfig.json

```diff
@@ -6,8 +6,9 @@
     "outDir": "./lib",
     "strict": true,
     "resolveJsonModule": true,
+    "allowJs": true,
```

Comment:
I added this to allow the .js usage test to run without warnings. I'm not sure if there's a less blanket way?

Reply:
Thanks for adding the test! I've added the js test to the `exclude` array in the `tsconfig`, which means we can remove the `allowJs` setting